app/helpers/stimulus_controller_attribute_helper.rb

```diff
@@ -0,0 +1,39 @@
+module StimulusControllerAttributeHelper
```

Comment:
Extracting these methods like this sounds very dependent on the technology we're using. I'm aware that there are very stimulus specific things going on here but I think I'd rename it to something more generic, like questions attribute helper etc. What do you think?